src/gatsby/src/components/park/visitResponsibly.js

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import { Link } from "gatsby"
+
+export default function VisitResponsibly({ campings, activities, marineProtectedArea }) {
+  const campingFacilityCodes = ["frontcountry-camping", "group-camping"]
+  const backcountryFacilityCodes = ["backcountry-camping", "wilderness-camping"]
+  const marineFacilityCode = ["marine-accessible-camping"]
+  const winterFacilityCode = ["winter-camping"]
+  const winterActivityCode = ["winter-recreation"]
+  const activityCodes = ["fishing", "hunting"]
+  const hasCamping = campings.some(c => campingFacilityCodes.includes(c.facilityType.facilityCode))
```

Comment:
these should probably check for nulls too `.includes(c.facilityType?.facilityCode)`